impl/src/main/java/com/okta/sdk/impl/http/support/SdkUserAgentProvider.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2018 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.impl.http.support;
+
+import com.okta.sdk.http.UserAgentProvider;
+
+/**
+ * @since 1.1.0
+ */
+public class SdkUserAgentProvider implements UserAgentProvider {
+
+    private static final String OKTA_SDK_STRING = "okta-sdk-java";
+    private static final String VERSION_SEPARATOR = "/";
```

Comment:
Since we're concatenating `OKTA_SDK_STRING` and `VERSION_SEPARATOR` below, why don't we make this as one variable?

Reply:
+1, this likely shouldn't even be a constant